docs/design/3_1_2_platform_management.tex

```diff
@@ -1 +1,52 @@
+\item provides an industry-standard camera interface (\textbf{F.CAM.2}),
+\item supports extensible non-volatile storage (\textbf{F.CP.2})\\
+\end{itemize} 
+
+\textbf{Data Flow}
+
+The overall data flow managed by the Raspberry Pi is as follows:
+\begin{enumerate}
+\item Video is acquired through the camera CSI interface
+\item Frame manipulation/correction is performed on the video, if required by the specific machine learning use-case
+\item Manipulated video is decomposed and transmitted over Ethernet to the programmable logic board (which performs the machine learning)
+\item Machine learning results are received via Ethernet
+\item Video, ML results, and system status information are transmitted via WiFi to the base station
+\item (Optional) Results and video are stored locally on non-volatile storage for post-flight analysis
+\end{enumerate}
+
+\textbf{Single-Board vs Two-Board Solution}
+
+The presence of on-board WiFi/Camera interfaces is the primary driving factor towards a two-board (Platform Management and Programmable) solution. While a single-board solution is possible, the development effort and cost of the computing platform would be considerably higher. Most programmable logic boards (including our chosen board, the Zedboard) require external expansion devices to interface with peripherals such as WiFi/cameras. These expansion devices, which connect to the programmable logic board via a serial connection, have poorly documented drivers and are of considerable expense -- ranging from \$25 USD for a basic WiFi module to \$200 USD for a compatible camera interface card. External expansion devices also require considerable glue logic resources on the programmable logic board, potentially risking non-compliance with constraint \textbf{C.EX.2}.
+
+An additional consideration in selecting the two-board design regards platform extensibility -- the client may desire to increase the Programmable Logic capabilities of the Computing Platform in the future, requiring a board swap. If a single-board solution is implemented, the client would be required to migrate \textit{all} hardware/software to the larger board, requiring them to devote considerable effort to debug driver issues. Additionally, as the external expansion devices are not necessarily compatible with all programmable logic boards -- adding constraints to the client's board selection or requiring the purchase of new interface cards. By offloading the interfacing work to a distinct \textit{platform management} board, the client can easily replace the PL board without considerable redevelopment (fulfilling constraint \textbf{C.EX.3}).
+
+\textbf{Interface with Programmable Logic Board}
+
+To interface with the programmable logic board, data packets are sent via a bi-directional TCP/IP-managed Ethernet connection. TCP/IP over Ethernet was selected in lieu of serial solutions (such as I2C) as it:
```

Comment:
```suggestion
To interface with the PLB, data packets are sent via a bi-directional TCP/IP-managed Ethernet connection. TCP/IP over Ethernet was selected in lieu of serial solutions (such as I2C) because:
```